airflow/contrib/hooks/gcp_dlp_hook.py

```diff
@@ -0,0 +1,1760 @@
+        :param template_id: (Optional) The template ID.
+        :type template_id: str
+        :param retry: (Optional) A retry object used to retry requests.
+            If None is specified, requests will not be retried.
+        :type retry: google.api_core.retry.Retry
+        :param timeout: (Optional) The amount of time, in seconds, to wait for the request
+            to complete. Note that if retry is specified, the timeout applies to each
+            individual attempt.
+        :type timeout: float
+        :param metadata: (Optional) Additional metadata that is provided to the method.
+        :type metadata: sequence[tuple[str, str]]]
+        :rtype: google.cloud.dlp_v2.types.InspectTemplate
+        """
+
+        client = self.get_conn()
+
+        if organization_id:
+            parent = DlpServiceClient.organization_path(organization_id)
+        elif project_id:
+            parent = DlpServiceClient.project_path(project_id)
+        else:
+            raise AirflowException(
+                "Please provide either organization_id or project_id."
+            )
+
+        return client.create_inspect_template(
+            parent=parent,
+            inspect_template=inspect_template,
+            template_id=template_id,
+            retry=retry,
+            timeout=timeout,
+            metadata=metadata,
+        )
+
+    @GoogleCloudBaseHook.catch_http_exception
+    @GoogleCloudBaseHook.fallback_to_default_project_id
+    def create_job_trigger(
+        self,
+        project_id=None,
+        job_trigger=None,
+        trigger_id=None,
+        retry=None,
+        timeout=None,
+        metadata=None,
+    ):
+        """
+        Creates a job trigger to run DLP actions such as scanning storage for sensitive
+        information on a set schedule.
+
+        :param project_id: (Optional) Google Cloud Platform project ID where the
+            DLP Instance exists. If set to None or missing, the default
+            project_id from the GCP connection is used.
+        :type project_id: str
+        :param job_trigger: (Optional) The job trigger to create.
+        :type job_trigger: dict or google.cloud.dlp_v2.types.JobTrigger
+        :param trigger_id: (Optional) The job trigger ID.
+        :type trigger_id: str
+        :param retry: (Optional) A retry object used to retry requests.
+            If None is specified, requests will not be retried.
+        :type retry: google.api_core.retry.Retry
+        :param timeout: (Optional) The amount of time, in seconds, to wait for the request
+            to complete. Note that if retry is specified, the timeout applies to each
+            individual attempt.
+        :type timeout: float
+        :param metadata: (Optional) Additional metadata that is provided to the method.
+        :type metadata: sequence[tuple[str, str]]]
+        :rtype: google.cloud.dlp_v2.types.JobTrigger
+        """
+
+        client = self.get_conn()
+
+        if not project_id:
```

Comment:
It looks like dead code, because decorator ``fallback_to_default_project_id`` prevent to execute this part of code